doc/design/abs_backup.md

```diff
@@ -0,0 +1,73 @@
+# Backups using Azure Blob Service (ABS)
+
+In a very similar fashion to backups using [S3](./s3_backup.md), we want to have an option to store backups in ABS.
+
+## Cluster configured with ABS backup
+
+To create a cluster with backups to ABS, we set `backup.storageType` to `"ABS"`.  Additionally, the ABS container for holding backups and the Kubernetes secret storing the Azure Storage account credentials must be supplied.  Both of these resources must exist prior to cluster creation (as will be discussed further below).
+
+An example cluster manifest would look like:
+
+```bash
+apiVersion: "etcd.database.coreos.com/v1beta2"
+kind: "EtcdCluster"
+metadata:
+  name: "etcd-cluster-with-abs-backup"
+spec:
+  ...
+  backup:
+    ...
+    storageType: "ABS"
+    abs:
+      absContainer: "myabscontainer"
+      absSecret: "abs-credentials"
+
+``` 
+
+### In Detail:
+
+- `"absSecret"` represents the name of the Kubernetes secret object that stores the environment variables needed for ABS storage account authorization, namely, the storage account name and the storage account key.
+
+  The Kubernetes secret manifest looks like:
+  ```
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: abs-credentials
+  type: Opaque
+  stringData:
+    storage-account: <storage-account-name>
+    storage-key: <storage-key>
+  ```
+
+  (Where the user is expected to provide the actual Azure Storage Account and Storage Key )
```

Comment:
Not sure if this line is necessary as it is quite obvious real account and key are required  

maybe 
```yaml
    storage-account: <your-storage-account-name>
    storage-key: <your-storage-key>
```
?